index.js

```diff
@@ -1,155 +0,0 @@
-games (); 
-async function games() {
-  let chooseGame = "d"; 
-  //Choose a game and the appropriate game will run 
-
-  while ((chooseGame.toLowerCase().trim() !== "a") && (chooseGame.toLowerCase().trim() !== "b") && (chooseGame.toLowerCase().trim() !== "c")) {
-  chooseGame = await ask(`Welcome! Please see the following menu and pick your game by entering the letter of your choice: 
-  a.) Play Computer Mind Reader - let the computer guess your secret number!
-  b.) Play Player Guesser Game - try to guess the computer's game!
-  c.) These games are lame and I am offended you think I would find them entertaining. Goodbye.\n`) 
-  if (chooseGame.toLowerCase().trim() == "a") {
-    start();
-  } else if (chooseGame.toLowerCase().trim() == "b") {
-    inverse(); 
-  } else if (chooseGame.toLowerCase().trim() == "c") {
-      process.exit() 
-    }
-  }
-  }
-
-async function start() { 
-
-  //opening message 
-  await ask(`\nWelcome to Computer Mind Reader! You will think of a number and I will use my superior clairvoyance to guess your secret number.
-  \nPress any key when you've thought of your number.\n`);
-
-  //starting values for max and min, declare min, max, guess variable 
-  let max = await ask("Give me a number that is higher than your number - this will be the maximum number that I will guess.\n");
-  max = +max;
-  let min = 1;
-  let guess = 0
-
-  //set up a way to keep track of guesses so can caluclate number of guesses 
-  let numbersGuessed = [];
-  let count = 0; 
-  function counter () {
-    count = numbersGuessed.length
-    return count 
-  }; 
-
-  //guesser function calculates a guess that is halfway between upper and lower bounds 
-  function guesser (max, min) {
-      guess =Math.floor((max + min)/2)
-      numbersGuessed.push(guess); 
-     return guess  
-  }
-  //calculate first guess
-  guesser(max,min);
-
-//start off yesOrNo as N to start the loop: 
-  let yesOrNo = "N"; 
-  while(yesOrNo.toUpperCase().trim() == "N") {
-
-  //ask if my guess is right or wrong 
-  yesOrNo = await ask(`Is your number ${guess}? Enter Y if yes or N if No.\n`);
-
-//if yes, game is over 
-if (yesOrNo.toUpperCase().trim() == "Y") {
-  console.log(`I KNEW your number was ${guess}!!!! And I guessed it in only ${count} tries!\n`);
-  
-  //want to play again? if so, let's go to main menu. if not, exit program 
-  let playAgain = await ask("That was F U N. Do you want to play another game? Type YES if you do\n");
-  if (playAgain.toUpperCase().trim() === "YES") { 
-    games (); 
-  }
-  else process.exit(); 
-}
-
-//if they didn't enter a Y or an N, try to ask about computer guess again 
-while ((yesOrNo.toUpperCase().trim() != "Y") && (yesOrNo.toUpperCase().trim() != "N")) {
-  yesOrNo = await ask(`You have to answer Y or N so I can know if I'm right!\n`); 
-}
-
-  //my guess is wrong so I need to figure out if higher or lower - start by defining variable and assigning to an unimportant value: 
-    let higherOrLower = "A" 
```

Comment:
If you wanted to create a variable to use that doesn't have an important value you can do `let higherOrLower;` and it will create the variable with nothing assigned to it.